### ISSUE TITLE
Service agreement not deploying

### ISSUE THREAD

**Issue by jernejpregelj** [oceanprotocol/keeper-contracts#264]
## Current Behavior

Starting Barge with spree network does not deploy 'Access service agreement template' and thus fails on all consume flows.

## Failure Information (for bugs)

```
brizo_1                      | 2019-01-09 10:20:15 5f30a9debebd root[10] INFO The `Access` Service agreement template "0x044852b2a670ade5407e78fb2863c51de9fcb96542a07186fe3aeda6bb8a116d" is not deployed to the current keeper network.
```

## Step to Reproduce the problem

Please provide detailed steps for reproducing the issue.

1. Run ```./start_ocean.sh --latest --local-spree-node```

**Comment by H34D** [comment]
thats more a problem of `keeper-contracts` transferring

**Comment by H34D** [comment]
Right now we are not deploying the Access template automatically. We could use this to do it: https://github.com/oceanprotocol/squid-js/blob/develop/src/examples/RegisterServiceAgreementTemplates.ts

**Comment by aaitor** [comment]
After a conversation with @H34D we agreed this logic to deploy the template needs to be part of the `keeper-contracts` deployment logic, making the template deployment independent of squid-js. Later we could discuss to move to an independent repo.

**Comment by H34D** [comment]
https://github.com/oceanprotocol/ocean/issues/262

**Comment by aaitor** [comment]
There is an update on this issue. Currently the keeper v0.8 deploy properly the template, but the template is not approval for the local or `spree` deployments because the multisig wallet.

A possible approach to fix this would be add an additional parameter to the `deployContracts.js` (i.e $KEEPER_LOCAL_DEPLOYMENT=true). If this environment variable exists and the value is `true`, the `owner` could be a simple accounts instead of the multisig wallet:

```
    const roles = {
        deployer: accounts[0],
        upgrader: accounts[1],
        initialMinter: accounts[2],
        owner: accounts[0],
        //owner: ownerWallet.address,
        admin: adminWallet.address
    }

```


In this scenario, from barge would be possible to setup this environment variable for `spree` deployment.

**Comment by Pedro-vk** [comment]
Could be useful be able to bind the owner address from outside. It is going to work only if has been defined as the environment variable.
E.g.: `OWNER_ROLE_ADDRESS="0xabc..."`
And then, in `deployContracts.js`:
```javascript
    const roles = {
        deployer: accounts[0],
        upgrader: accounts[1],
        initialMinter: accounts[2],
        owner: process.env.OWNER_ROLE_ADDRESS || ownerWallet.address,
        admin: adminWallet.address
    }
```
It is going to allow to deploy any template on development with different web3 providers (not only HDWallet).
What do you think @H34D ?

**Comment by H34D** [comment]
> It is going to allow to deploy any template on development with different web3 providers (not only HDWallet).
> What do you think @H34D ?

We could do the following:
- Create a new role `templateAdmin` that is not multi sig
- Change the deployment to approve this specific template before assigning the multi sig wallet
- Use the multisig process to approve this template during the deployment process

The current version of the deployment works different from what this discussion is based on. Let's do requirements engineering first then think about the solution. For now the workaround from #400 is ok, we need a different solution for production.

**Comment by H34D** [comment]
closes with #394

**Comment by H34D** [comment]
A workaround has been implemented to skip multi sig wallet usage. Approving this automatically still needs to be implemented.

**Comment by H34D** [comment]
closes with #412
